tests/builtins/test_isinstance.py

```diff
@@ -70,6 +67,24 @@ class MyClass2(MyClass1):
             print(isinstance(obj, MyClass2))
             """, run_in_function=False)
 
+    def test_subclass_grandparent(self):
```

Comment:
Hmm, there is is a `issubclass(MyClass3, MyClass1)` at [`test_issubclass.py:75`](https://github.com/pybee/voc/blob/31e0cee28f9e6ff7b4c8cdcacf9ed3cae5d1b222/tests/builtins/test_issubclass.py#L75).

Here I was following the previous `test_subclass()` test case, but I think I will rename them to `test_parent()` and `test_grandparent()` instead.